src/shogun2-core/shogun2-model/src/main/java/de/terrestris/shogun2/model/interceptor/InterceptorRule.java

```diff
@@ -0,0 +1,225 @@
+package de.terrestris.shogun2.model.interceptor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+import de.terrestris.shogun2.model.PersistentObject;
+
+/**
+ * The model representing the rules for the GeoServer Interceptor class.
+ *
+ * A rule is following the schema:
+ *
+ * <event>.<service>.<operation>.<endPoint>=<rule>
+ *
+ * The rules will be evaluated for every request where we determine the most
+ * specific rule to apply.
+ *
+ * Allowed values for the rule are ALLOW, DENY, MODIFY.
+ * Allowed values for the event are REQUEST and RESPONSE.
+ *
+ * @author Daniel Koch
+ * @author Kai Volland
+ * @author terrestris GmbH & Co. KG
+ *
+ */
+@Entity
+@Table
+public class InterceptorRule extends PersistentObject {
+
+	/**
+	 *
+	 */
+	private static final long serialVersionUID = 1L;
+
+	/**
+	 * The event for this rule, possible values are:
+	 *   * REQUEST
+	 *   * RESPONSE
+	 */
+	@Column(nullable = false)
+	private String event;
+
+	/**
+	 * The rule type for this rule, possible rules are:
+	 *   * ALLOW
+	 *   * DENY
+	 *   * MODIFY
+	 */
+	@Column(nullable = false)
+	private String rule;
+
+	/**
+	 * The OGC service type, e.g. WMS, WFS or WCS.
+	 */
+	@Column(nullable = false)
+	private String service;
+
```

Comment:
Minor: Maybe it would be more "clean" to use an Enum here (for example as we do [here](https://github.com/terrestris/shogun2/blob/master/src/shogun2-core/shogun2-model/src/main/java/de/terrestris/shogun2/model/module/NominatimSearch.java#L46))